src/mem/allocstats.h

```diff
@@ -387,15 +387,15 @@ namespace snmalloc
         sizeclass[i].print(csv, sizeclass_to_size(i));
       }
 
-      for (uint8_t i = 0; i < LARGE_N; i++)
-      {
-        if ((large_push_count[i] == 0) && (large_pop_count[i] == 0))
-          continue;
+      // for (uint8_t i = 0; i < LARGE_N; i++)
+      // {
+      //   if ((large_push_count[i] == 0) && (large_pop_count[i] == 0))
+      //     continue;
 
-        csv << "LargeBucketedStats" << dumpid << allocatorid << (i + N)
-            << large_sizeclass_to_size(i) << large_push_count[i]
-            << large_pop_count[i] << csv.endl;
-      }
+      //   csv << "LargeBucketedStats" << dumpid << allocatorid << (i + N)
+      //       << large_sizeclass_to_size(i) << large_push_count[i]
+      //       << large_pop_count[i] << csv.endl;
+      // }
```

Comment:
Is this being left in to be added back later?

Reply:
Yeah, the whole alloc_stats bit needs rewriting. I was trying not to delete code, but didn't want to fail compilation for this.